.travis.yml

```diff
@@ -22,9 +21,14 @@ install:
       - conda update -q conda
         # Useful for debugging any issues with conda
       - conda info -a
-      - conda install -q --yes --file requirements.txt -c cta-observatory -c conda-forge -c openastronomy
-      - python setup.py install
+      - conda create -n test python=3.6
+      - source activate test
+      - conda config --add channels anaconda
+      - conda config --add channels conda-forge
+      - conda config --add channels cta-observatory
+      - conda install pytest-cov
+      - conda install -c bryankim96 dl1_data_handler
```

Comment:
This breaks the travis build.

This will only ever test the latest release version on conda. Not the code in Pull Requests or the current branch or the master.

Installing anything you can using conda is fine, but the code under test must be installed locally using pip, or made into a conda package locally using `conda build` and `conda install --local`.

Reply:
Replace with `pip install .`

Reply:
Thanks for the catch! I didn't think about this.